env0/resource_environment_output_configuration_variable_test.go

```diff
@@ -0,0 +1,298 @@
+package env0
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"strconv"
+	"testing"
+
+	"github.com/env0/terraform-provider-env0/client"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"go.uber.org/mock/gomock"
+)
+
+func TestUnitEnvironmentOutputConfigurationVariableResource(t *testing.T) {
+	resourceType := "env0_environment_output_configuration_variable"
+	resourceName := "test"
+	resourceNameImport := resourceType + "." + resourceName
+
+	accessor := resourceAccessor(resourceType, resourceName)
+
+	value := EnvironmentOutputConfigurationVariableValue{
+		OutputName:    "output_name",
+		ProjectId:     "output_project_id",
+		EnvironmentId: "output_environment_id",
+	}
+
+	valueBytes, _ := json.Marshal(&value)
+	valueStr := string(valueBytes)
+
+	updatedValue := EnvironmentOutputConfigurationVariableValue{
+		OutputName:    "output_name2",
+		ProjectId:     "output_project_id2",
```

Comment:
please remove the project ids from all places including tests

Reply:
Got it!

Reply:
@tomer-landesman - this is now done.
Harness tests were updated as well.